com.avaloq.tools.ddk.xtext.ui.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,14 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: com.avaloq.tools.ddk.xtext.ui.test
+Bundle-SymbolicName: com.avaloq.tools.ddk.xtext.ui.test;singleton:=true

```

Comment:
@rmitin, do we still do test plugins as fragments?

Reply:
Some DDK test projects do. Some (including check.ui.test and checkcfg.ui.test) do not. I shall await the wisdom of @rmitin .

Reply:
@dpetroff , we do unit tests as fragments and integration tests as standalone bundles. Most DSL tests that test languages are implemented as fragments and we treat them as unit tests. I am not aware why for check they were implemented as plugins. If tests are more like integration tests, then they are done with plugins. 

Unit tests typically do not introduce many additional dependencies to the host bundle (except for dependencies to testing frameworks) and therefore are possible as fragments. This way they also get access to internal implementation details of the host bundle. So it might not be possible to write such unit tests as a separate plugin. If tests here are more like integration tests and will pull additional dependencies into the manifest, rather keep them as a plugin. It might even not be possible to do it as a fragment (i.e. running into circular dependencies between the host bundle bundles required by the test fragment).

Reply:
The project those tests were moved from (com.avaloq.tools.foundation.xtext.ui.test) isn't a fragment; since I'm merely moving that functionality and test coverage into DDK, I propose using the same style of test project.